yt_dlp/__init__.py

```diff
@@ -436,19 +436,9 @@ def metadataparser_actions(f):
             default_downloader = ed.get_basename()
 
     if opts.color:
-        policies = {
-            'always': True,
-            'auto': None,
-            'no_color': 'no_color',
-            'never': False,
-        }
-        result = {}
-        for name, policy in opts.color.items():
-            if policy not in policies:
+        for policy in opts.color.values():
+            if policy not in ('always', 'auto', 'no_color', 'never'):
                 raise ValueError(f'"{policy}" is not a valid color policy')
```

Comment:
My suggestion used items so we can write `"{policy}" is not a valid color policy for {name}`. Not a big deal, but idk why you changed it.

Reply:
We don't want to output name since it might be confusing, `--color stdour:never` will print `"stdour:never" is not a valid color policy for stdout` otherwise.